po/vi.po

```diff
@@ -2742,25 +2742,37 @@ msgid ""
 "this course. Try running the \"hello-world\" program it starts with. It "
 "comes with a few handy features:"
 msgstr ""
+"[Nơi thử nghiệm Rust (*Rust Playground*)](https://play.rust-lang.org/) cung "
+"cấp một cách dễ dàng để chạy các chương trình Rust ngắn, và là cơ sở cho các "
+"ví dụ và bài tập trong khóa học này. Hãy thử chạy chương trình “hello-"
+"world”. Nó đi kèm với một vài tính năng hữu ích:"
 
 #: src/hello-world/playground.md
 msgid ""
 "Under \"Tools\", use the `rustfmt` option to format your code in the "
 "\"standard\" way."
 msgstr ""
+"Ở phần “Công cụ (*Tools*)”, sử dụng tuỳ chọn`rustfmt` để định dạng đoạn code "
+"của bạn theo cách “tiêu chuẩn”."
 
 #: src/hello-world/playground.md
 msgid ""
 "Rust has two main \"profiles\" for generating code: Debug (extra runtime "
 "checks, less optimization) and Release (fewer runtime checks, lots of "
 "optimization). These are accessible under \"Debug\" at the top."
 msgstr ""
+"Rust có hai “chế độ” chính để tạo code: Gỡ lỗi (*Debug*) (kiểm tra thêm thời "
+"gian chạy (*runtime*), ít tối ưu hơn) và Phát hành (*Release*) (ít kiểm tra "
+"thời gian chạy (*runtime*) hơn, tối ưu hoá nhiều hơn). Các tuỳ chọn này có "
+"trong mục “Gỡ lỗi (*Debug*)” ở phía trên."
```

Comment:
`extra runtime checks` should be translated to `thêm nhiều thông tin hỗ trợ trong quá trình chạy`. The same should be applied for `fewer runtime checks`